.github/workflows/build.yml

```diff
@@ -53,5 +53,5 @@ jobs:
           key: maven-repo-${{ runner.os }}-${{ steps.get-date.outputs.date }}
 
       - name: Build with Maven
-        run: mvn -B formatter:validate verify --file pom.xml
+        run: mvn -B formatter:validate verify -Dnative --file pom.xml
```

Comment:
This should be done in a different job, so that we can run both native and jvm tests to make sure we cover everything